Documentation/fw_dell_optiplex.md

```diff
@@ -32,7 +32,7 @@ correctly.
     docker run --rm -it \
         -v $PWD:/home/coreboot/coreboot \
         -w /home/coreboot/coreboot \
-        coreboot/coreboot-sdk:0ad5fbd48d
+        coreboot/coreboot-sdk:2023-06-04_44f676afc9
     ```
 
 - Build coreboot firmware:
```

Comment:
This builds without errors, but I don't think `Built dell/e6400 (Latitude E6400)` is the expected result...

Reply:
Okay... New target landed upstream...